mmtrack/models/trackers/byte_tracker.py

```diff
@@ -0,0 +1,313 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import lap
+import numpy as np
+import torch
+from mmcv.runner import force_fp32
+from mmdet.core import bbox_overlaps
+
+from mmtrack.models import TRACKERS
+from .base_tracker import BaseTracker
+
+
+@TRACKERS.register_module()
+class ByteTracker(BaseTracker):
+    """Tracker for ByteTrack.
+
+    Args:
+        high_det_score (float): Threshold of the first matching detection bbox.
```

Comment:
We used `obj_score_thr` in SORT and Tracktor. How about `obj_score_lthr` and `obj_score_hthr` here?
Or we can use `obj_score_thrs = [A, B]` or even use a `dict`. 

If you want `det_score_thr`, which is also reasonable, better to revise the other methods (but is BC-breaking : ( ).

Reply:
I'll name it as `obj_score_low_thr` and `obj_score_high_thr`, since the `obj_score_lthr` and `obj_score_hthr` are indistinguishable.